### DISCUSSION TITLE
Welcome to nrsc5-dui Discussions!

### DISCUSSION THREAD

**Discussion by markjfine** [markjfine/nrsc5-dui#24] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.
  * A place for general discussion that aren't software issues.

  To get started, comment below with an introduction of yourself and tell us about what you would like to see done with this community.

Note: Many of the open items that were in Issues but were open-ended discussions have been moved here to keep the Issues area more focused on general enhancements and bugs.

**Comment by Sifti10** [comment]
I have been using an RTL-DSR on both windows and android. I could not tap into the HD radio signals with any of the software available. I finally found your information. I am not too familiar with Linux. I installed it on an old PC to try nrsc5-dui. I am having trouble getting it to work. My knowledge of using github is pretty basic as well. Can you provide a more step by step procedure?

**Reply by markjfine** [reply]
The readme specifically says it doesn't work on Windows at the current time.
